src/SourceBuild/content/repo-projects/installer.proj

```diff
@@ -10,6 +10,8 @@
 
     <!-- Restore and Build actions are already passed in by the root script. -->
     <BuildActions>$(FlagParameterPrefix)pack $(FlagParameterPrefix)publish</BuildActions>
+    
+    <SkipEnsurePackagesCreated>true</SkipEnsurePackagesCreated>
```

Comment:
Because installer doesn't produce any package output. Without this flag, it would cause a build error from this: https://github.com/dotnet/installer/blob/162e7d45bc95233f9902a6746841053348a47185/src/SourceBuild/content/repo-projects/Directory.Build.targets#L570-L571

This wasn't needed before because that check was checking whether _any_ packages exist in the output location. So it was finding packages that came from other repos. Now that everything is isolated, it's only checking for packages that were output by installer and there are none, as expected.